handler/routes.go

```diff
@@ -358,17 +362,17 @@ func GetClient(db store.IStore) echo.HandlerFunc {
 
 		clientID := c.Param("id")
 		qrCodeSettings := model.QRCodeSettings{
-			Enabled:       true,
-			IncludeDNS:    true,
-			IncludeMTU:    true,
+			Enabled:    true,
```

Comment:
Drop changes like this. They can be fixed later. They only add noise to this PR.

Reply:
In a separate commit, in a separate PR.

It's default gofmt. It should be this way anyway.

Reply:
I've made PRs which waited for years, even tho the repo is active. But reviewers don't have unlimited time to pick apart your PR. The first question they ask themselves is: how does this formatting affect this new functionality? Go is a bit weird where indentation is pedantic.

Don't conflate fixing someone else's formatting omission with your PR.


Once the PR gets accepted, do a new one with go fmt results if the repo owner doesn't do this themselves.